platforms/hyperledger-fabric/configuration/roles/create/ca-tools/orderer/tasks/main.yaml

```diff
@@ -115,25 +115,81 @@
   when: network.env.proxy == 'ambassador'
 
 ############################################################################################
-# This task copies the msp folder from the respective CA Tools CLI
-# to the Ansible container
-- name: Copy the msp folder from the ca tools
+# This task copies the msp admincerts from vault
+- name: Copy the msp admincerts from vault
```

Comment:
Please rename the task and the above comment to `Fetch the msp admincerts from vault`